crates/rattler_installs_packages/src/wheel_builder/mod.rs

```diff
@@ -150,7 +150,9 @@ impl<'db, 'i> WheelBuilder<'db, 'i> {
             env_markers,
             wheel_tags,
             resolve_options,
-            locally_built_wheels: WheelCache::new(wheel_cache_dir.to_path_buf()),
+            locally_built_wheels: WheelCache::new(
+                wheel_cache_dir.to_path_buf().join("wheel_cache"),
```

Comment:
Should this be more descriptive?
One thing I dislike is that the user doesn't have full control over this location now.

Reply:
Yeah, maybe? Would that work?

Reply:
Okay @wolfv, changed it, does this better reflect what you want?

I've made it move by value as the cache is not really an object but more or less just wraps the location that the cache is in, this is how cacache works.

What I think is the downside is that it is easy to forget to add the `join('wheels')`. We could also add the `WheelCache` or path to the `PackageDb`. WDYT?